src/main/app-updater.ts

```diff
@@ -9,9 +9,8 @@ function handleAutoUpdateBackChannel(event: Electron.IpcMainEvent, ...[arg]: Upd
   if (arg.doUpdate) {
     if (arg.now) {
       logger.info(`${AutoUpdateLogPrefix}: User chose to update now`);
-      autoUpdater.downloadUpdate()
-        .then(() => autoUpdater.quitAndInstall())
-        .catch(error => logger.error(`${AutoUpdateLogPrefix}: Failed to download or install update`, { error }));
+      autoUpdater.on("update-downloaded", () => autoUpdater.quitAndInstall());
```

Comment:
Not sure how it can fix it when in `app-updater.ts`:

```ts  
  // Whether to automatically install a downloaded update on app quit (if quitAndInstall was not called before). Applicable only on Windows and Linux.
  autoUpdater.autoInstallOnAppQuit = false; 
  ```

@jakolehm how it actually fixing the issue? `autoUpdater.quitAndInstall().then(quitAndInstall)` is never called? 
or how it differs from `autoUpdater.on("update-downloaded", quitAndInstall)` event-call?

Reply:
`autoUpdater.quitAndInstall().then(quitAndInstall)` calls too fast (it does not find the update from filesystem). Docs say that it should be called _after_ `update-downloaded` event.